charts/consul/values.yaml

```diff
@@ -3098,6 +3119,22 @@ terminatingGateways:
   # @type: string
   logLevel: ""
 
+  # This configures the [`PodDisruptionBudget`](https://kubernetes.io/docs/tasks/run-application/configure-pdb/)
+  # for the terminating gateways.
+  disruptionBudget:
+    # Enables registering a PodDisruptionBudget for the terminating gateways
+    # If enabled, it only registers the budget so long as
+    # the terminating gateways is enabled. To disable, set to `false`.
+    enabled: true
+
+    # The maximum number of unavailable pods. By default, this will be
+    # automatically computed based on the `server.replicas` value to be `(n/2)-1`.
+    # If you need to set this to `0`, you will need to add a
+    # --set 'server.terminatingGateways.maxUnavailable=0'` flag to the helm chart installation
+    # command because of a limitation in the Helm templating language.
```

Comment:
> command because of a limitation in the Helm templating language.

What limitation is this?

Reply:
ah, seems to be a copied comment not applicable here